src/main/java/com/hotjava/app/hotjava/dao/VehicleDAOStub.java

```diff
@@ -17,14 +17,18 @@
 
     @Override
     public Vehicle save(Vehicle vehicle) throws Exception {
-        allVehicles.put(vehicle.getVehicleSubmissionID(), vehicle);
+        try {
+            allVehicles.put(vehicle.getSubmissionID(), vehicle);
+        }
+        catch (Exception e) {
```

Comment:
If the method throws an exception on failure, handle the exception within a catch block.

Reply:
This is worse off than before.  You're catching the exception, then ignoring it, and then acting as if everything worked fine on line 26.  That hides very important information about what went wrong.

If adding a catch block, heal the exception there.  At the very least, add a logging statement.  An empty catch block is the worst possible option.